frameworks/g-cloud-12/metadata/copy_services.yml

```diff
@@ -0,0 +1,162 @@
+- dataSanitisation
+- dataSanitisationType
+- dataStorageAndProcessing
+- dataStorageAndProcessingLocations
+- dataStorageAndProcessingUserControl
+- devicesUsersManageTheServiceThrough
+- educationPricing
+- endOfContractDataExtraction
+- endOfContractProcess
+- equipmentDisposalApproach
+- freeVersionDescription
+- freeVersionLink
+- freeVersionTrialOption
+- gettingStarted
+- governmentSecurityClearances
+- governmentSecurityClearancesFilter
+- guaranteedAvailability
+- howLongSystemLogsStored
+- incidentManagementApproach
+- incidentManagementType
+- independenceOfResources
+- informationSecurityPoliciesAndProcesses
+- installation
+- installationCompatibleOperatingSystems
+- lot
+- lotName
+- metrics
+- metricsHow
+- metricsWhat
+- metricsWhatOther
+- metricsDescription
+- mobile
+- mobileDifferences
+- ongoingSupport
+- ongoingSupportDescription
+- ongoingSupportServices
+- onsiteSupport
+- outageReporting
+- penetrationTesting
+- penetrationTestingApproach
+- phoneSupport
+- phoneSupportAvailability
+- planningService
+- planningServiceCompatibility
+- planningServiceCompatibilityList
+- planningServiceDescription
+- priceInterval
+- priceMax
+- priceMin
+- priceUnit
+- protectionOfDataAtRest
+- protectionOfDataAtRestOther
+- protectiveMonitoringApproach
+- protectiveMonitoringType
+- QAAndTesting
+- QAAndTestingDescription
+- resellingOrganisations
+- resellingType
+- scaling
+- scalingType
+- secureDevelopment
+- securityGovernanceAccreditation
+- securityGovernanceApproach
+- securityGovernanceStandards
+- securityGovernanceStandardsOther
+- serviceAddOnDetails
+- serviceAddOnType
+- serviceBenefits
+- serviceCategories
+- serviceConstraints
+- serviceDescription
+- serviceFeatures
+- serviceName
+- setupAndMigrationService
+- setupAndMigrationServiceDescription
+- setupAndMigrationServiceSpecific
+- setupAndMigrationServiceSpecificList
+- staffSecurityClearanceChecks
+- standardsCSASTAR
+- standardsCSASTARExclusions
+- standardsCSASTARLevel
+- standardsCSASTARWhen
+- standardsISO28000
+- standardsISO28000Exclusions
+- standardsISO28000When
+- standardsISO28000Who
+- standardsISOIEC27001
+- standardsISOIEC27001Exclusions
+- standardsISOIEC27001When
+- standardsISOIEC27001Who
+- standardsPCI
+- standardsPCIExclusions
+- standardsPCIWhen
+- standardsPCIWho
+- supportAvailableToThirdParty
+- supportLevels
+- systemRequirements
+- training
+- trainingDescription
+- trainingServiceSpecific
+- trainingServiceSpecificList
+- usageNotifications
+- usageNotificationsHow
+- userAuthenticationDescription
+- userAuthentication
+- userAuthenticationNeeded
+- virtualisation
+- virtualisationImplementedBy
+- virtualisationSeparation
+- virtualisationTechnologiesUsed
+- virtualisationTechnologiesUsedOther
+- virtualisationThirdPartyProvider
+- vulnerabilityManagementApproach
+- vulnerabilityManagementType
+- webInterface
+- webInterfaceAccessibility
+- webInterfaceAccessibilityDescription
+- webInterfaceAccessibilityTesting
+- webInterfaceUsage
```

Comment:
Some fields were manually excluded from the G11 `copy_services` list as they had changed since G10. We can now add them back in (the script isn't yet clever enough to parse all the available questions):

```
  - cloudDeploymentModel
  - documentation
  - documentationFormats
  - documentationFormatsOther
  - emailOrTicketingSupport
  - emailOrTicketingSupportAccessibility
  - emailOrTicketingSupportPriority
  - emailOrTicketingSupportResponseTimes
  - energyEfficientDatacentres
  - managementAccessAuthentication   # This one caused the copying bug last time so let's be careful to test it
  - managementAccessAuthenticationDescription
  - serviceInterfaceAccessibility
  - serviceInterfaceAccessibilityDescription
  - serviceInterfaceTesting
  - webChatSupport
  - webChatSupportAccessibility
  - webChatSupportAccessibilityDescription
  - webChatSupportAccessibilityTesting
  - webChatSupportAvailability
```

Reply:
Added those back in 👍